socorro/unittest/external/es/test_connection_context.py

```diff
@@ -15,8 +15,11 @@
 
 class TestConnectionContext:
     def test_create_index(self, es_conn):
-        # Verify there are no indices first
-        assert list(es_conn.get_indices()) == []
+        # Delete any existing indices first
+        for index in es_conn.get_indices():
+            es_conn.delete_index(index)
+        es_conn.refresh()
+        es_conn.health_check()
```

Comment:
Because I changed `es_conn` to create two indexes, we have to delete them for this test to run.